package.json

```diff
@@ -55,7 +55,7 @@
   },
   "dependencies": {
     "@isaacs/string-locale-compare": "^1.1.0",
-    "@npmcli/arborist": "^4.1.1",
+    "@npmcli/arborist": "github:nishnha/arborist#dependabot-audit",
```

Comment:
should be able to revert this change since arborist is a workspace in this repo now